<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transformation</class>
 <widget class="QWidget" name="Transformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>328</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transformation</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>style</string>
   </property>
  </widget>
  <widget class="LSlider" name="MoveXSlider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>22</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="LSlider" name="MoveYSlider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>22</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="LSlider" name="RotateSlider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>20</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>-180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>241</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>52</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>upward </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>241</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>52</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>down</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>241</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>52</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>guide</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Rotate">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>31</y>
     <width>64</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rotation Angle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RotateEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>31</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="MoveX">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>71</y>
     <width>64</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MoveX_2">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>71</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="MoveY">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>111</y>
     <width>64</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MoveYEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>111</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="FlipModel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>151</y>
     <width>70</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>1000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Flip</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="LcomboBox" name="ModelBox">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>151</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>vertical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>horizontal</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>201</y>
     <width>35</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>201</y>
     <width>35</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LineButton">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>201</y>
     <width>35</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="WidthEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>40</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft New Tai Lue</family>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>211</y>
     <width>40</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LSlider</class>
   <extends>QSlider</extends>
   <header location="global">LSlider.h</header>
  </customwidget>
  <customwidget>
   <class>LcomboBox</class>
   <extends>QComboBox</extends>
   <header location="global">LcomboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
